src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -27,6 +28,10 @@ class Vehicle(val id: Int, val wannaDriveInHours: MutableList<Int>) {
         if (!wannaDriveInHours.contains(timestep)) {
             delay -= 1
         }
+
+        if (delay <= 0 && wannaDriveInHours.contains(timestep)){
```

Comment:
Die Bedingung verstehe ich nicht. Das Netzwerk informiert in dieser Methode das Auto darüber, dass gefahren wird. Da muss doch in jedem Fall vermerkt werden, dass es gefahren ist.